exercises/pig-latin/pig_latin_test.exs

```diff
@@ -63,7 +63,7 @@ defmodule PigLatinTest do
 
     @tag :pending
     test "word beginning with q without a following u" do
-      assert PigLatin.translate("qat") == "atqay"
+      assert PigLatin.translate("qatar") == "atarqay"
```

Comment:
"qat" is a real word (a type of bush, I think). But this also works.

Reply:
Nice :+1:. You can tell I'm not much of a Scrabble player.

Reply:
Sadly, I suq at Scrabble.